src/DotNetWorker.Core/Converters/Converter/AllowConverterFallbackAttribute.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.Azure.Functions.Worker.Converters
+{
+    /// <summary>
+    /// An attribute that specifies if Converter fallback is allowed
+    /// </summary>
+    [AttributeUsage(
+        AttributeTargets.Parameter |
+        AttributeTargets.Class |
+        AttributeTargets.Interface |
+        AttributeTargets.Enum |
+        AttributeTargets.Struct)]
+    public sealed class AllowConverterFallbackAttribute : Attribute
+    {
+        /// <summary>
+        /// Gets the value of whether Converter fallback is allowed.
+        /// </summary>
+        public bool AllowConverterFallback { get; }
+
+        /// <summary>
+        /// Creates a new instance of <see cref="AllowConverterFallbackAttribute"/>
+        /// </summary>
+        /// <param name="allowConverterFallback">The value to indicate if converter fallback is allowed.</param>
+        public AllowConverterFallbackAttribute(bool allowConverterFallback)
```

Comment:
It looks like the default if this attribute isn't specified is to allow fallback? I.e. the attribute isn't required. Then I wonder why you have an attribute that allows specification of true/false, and instead don't just have an attribute to disable fallback when needed (default being true). E.g. [DisableConverterFallback].

Reply:
We have the `true`/`false` because we want to add an analyzer to suggest all input binding implementations that use `SupportsDeferredBinding` to be explicit about their `AllowConverterFallback` behavior. Thus, we need to be able to specify both `true` or `false`.

Reply:
We did have discussion for both AllowConverterFallback and DisableConverterFallback in our design syncs. From last meeting, I remember we wanted to use AllowConverterFallback, unless I am missing anything. @jviau do you have more details/preference to share on this?